src/style/CharitySection.less

```diff
@@ -0,0 +1,15 @@
+
+@import "../js/base/style/fonts.less";
+
+.charity-section {
+    h2 {
+        color: black !important;
+        font-family: 'Montserrat', sans-serif;
+        font-weight: normal;
+        font-size: 2.5rem;
```

Comment:
@wongws11 Are the font settings not defined in fonts?

Having section-specific font css is _usually_ to be avoided, because it easily leads to inconsistent fonts.

Reply:
NB: When you do want local font settings (which happens), leave a comment as to why.

Reply:
It is an override to the colour, weight and size to match the new design. I will give a comment explaining this.

Reply:
Hello @wongws11 I can see _what_ it does (colour, weight, family) . What's unclear is _why_ we want local overrides for these things. I.e. why is the font for this section different from the font settings defined more generally and used elsewhere?

If the answer is "That's what's in the design I have", then we should ask: Should it be different here? 

Sometimes, a designer will work on one page and use a particular font setup. But they don't actually want it to be _different_ from the other pages, it's just that they were only looking at that page.

Reply:
@wongws11 Hello Wing - I think this question is all that's blocking the PR, so please respond on this ASAP. Thanks.

Reply:
I get your point now. I use this local font settings because I want to match the exact design from Kati. But now that I know we should stick to the global design if we have no other special reasons. I reckon we should not use any local css here, so the title will match the overall website design.